libs/platform/src/lib/components/dynamic-page/dynamic-page.component.ts

```diff
@@ -266,22 +268,24 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
      * set styles for tab labels
      */
     private _setTabStyles(): void {
-        const tabList = this._elementRef.nativeElement.querySelector('.fd-tabs');
-        if (tabList) {
-            this._addClassNameToCustomElement(tabList, CLASS_NAME.dynamicPageTabs);
-            this._addClassNameToCustomElement(tabList, CLASS_NAME.dynamicPageTabsAddShadow);
-            if (this.size) {
-                this._setTabsSize(this.size, tabList);
-            }
-            if (this.headerComponent?.collapsible) {
-                const pinCollapseShadowElement = this._elementRef.nativeElement.querySelector(
-                    '.fd-dynamic-page__collapsible-header-visibility-container'
-                );
-                if (pinCollapseShadowElement) {
-                    this._addClassNameToCustomElement(
-                        pinCollapseShadowElement,
-                        CLASS_NAME.dynamicPageCollapsibleHeaderPinCollapseNoShadow
+        if (this.contentContainer) {
+            const tabList = this.contentContainer.nativeElement.querySelector('.fd-tabs');
+            if (tabList) {
+                this._addClassNameToCustomElement(tabList, CLASS_NAME.dynamicPageTabs);
+                this._addClassNameToCustomElement(tabList, CLASS_NAME.dynamicPageTabsAddShadow);
+                if (this.size) {
+                    this._setTabsSize(this.size, tabList);
+                }
+                if (this.headerComponent?.collapsible) {
+                    const pinCollapseShadowElement = this._elementRef.nativeElement.querySelector(
+                        '.fd-dynamic-page__collapsible-header-visibility-container'
                     );
+                    if (pinCollapseShadowElement) {
+                        this._addClassNameToCustomElement(
+                            pinCollapseShadowElement,
+                            CLASS_NAME.dynamicPageCollapsibleHeaderPinCollapseNoShadow
+                        );
+                    }
                 }
             }
         }
```

Comment:
the same
decrease nesting
```suggestion
        if (!this.contentContainer) {
            return;
        }

        const tabList = this.contentContainer.nativeElement.querySelector('.fd-tabs');
        if (!tabList) {
            return;
        }

        this._addClassNameToCustomElement(tabList, CLASS_NAME.dynamicPageTabs);
        this._addClassNameToCustomElement(tabList, CLASS_NAME.dynamicPageTabsAddShadow);
        if (this.size) {
            this._setTabsSize(this.size, tabList);
        }

        if (!this.headerComponent?.collapsible) {
            return;
        }

        const pinCollapseShadowElement = this._elementRef.nativeElement.querySelector(
            '.fd-dynamic-page__collapsible-header-visibility-container'
        );
        if (!pinCollapseShadowElement) {
            return;
        }

        this._addClassNameToCustomElement(
            pinCollapseShadowElement,
            CLASS_NAME.dynamicPageCollapsibleHeaderPinCollapseNoShadow
        );
```

Reply:
fixed as suggested, except the last if `if (!pinCollapseShadowElement) {
            return;
        }`  was just kept like before since it wasn't further nested anyway =>
```
if (pinCollapseShadowElement) 
{ 
         this._addClassNameToCustomElement(
            pinCollapseShadowElement,
            CLASS_NAME.dynamicPageCollapsibleHeaderPinCollapseNoShadow
        ); 
}